src/Umbraco.Cms.Integrations.PIM.Inriver/App_Plugins/UmbracoCms.Integrations/PIM/Inriver/Render/InriverEntity.cshtml

```diff
@@ -1,15 +1,18 @@
 ﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Umbraco.Cms.Integrations.PIM.Inriver.Models.ViewModels.InriverEntityViewModel>
 
 <div>
-    <img src="@Model.ResourceUrl" alt="@Model.ResourceUrl" />
+    <img src="@Model.ResourceUrl" alt="@Model.ResourceUrl" width="100" />
     <p>@Model.DisplayName</p>
     <p>@Model.DisplayDescription</p>
     @if (Model.Fields.Any())
     {
         <ul>
             @foreach (var field in Model.Fields)
             {
-                <li><b>@field.Key</b>: @field.Value</li>
+                <li>
+                    <b>@field.FieldTypeId</b>: @field.Value
+                    @field.Display
```

Comment:
Maybe you should apply a bit more mark-up around this, so if someone wants to use this rendering component they could do so, and would have some hooks for styling.